src/libmongoc/src/mongoc/mongoc-crypto-openssl.c

```diff
@@ -39,13 +39,13 @@ mongoc_crypto_openssl_hmac_sha1 (mongoc_crypto_t *crypto,
 }
 
 #if OPENSSL_VERSION_NUMBER < 0x10100000L || (defined(LIBRESSL_VERSION_NUMBER) && LIBRESSL_VERSION_NUMBER < 0x20700000L)
-EVP_MD_CTX *
+static EVP_MD_CTX *
```

Comment:
@remicollet: Can you add `static` declarations to the polyfills in https://github.com/mongodb/mongo-c-driver/blob/1.21.0/src/kms-message/src/kms_crypto_libcrypto.c#L25 as well?

Reply:
@jmikola hmm... it is already there ? or I'm blind ?